contrib/ci/test-swift.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
```

Comment:
I feel like this should be broken down into several parts so we don't have duplicate code lying around in the integration tests.